Content/1.1 - Transaction Review/transactionReview.json

```
@@ -18,6 +18,21 @@
 	"transactionReview_networkFee_heading": "Transaction Fee",
 	"transactionReview_networkFee_congestedText": "The network is currently congested. Add a tip to speed up your transfer.",
 	"transactionReview_networkFee_customizeButtonTitle": "Customize",
+	"transactionReview_networkFeeSheet_default_title": "Customize Fees",
```

I would change to `default` to `normalMode`
```suggestion
	"transactionReview_networkFeeSheet_normalMode_title": "Customize Fees",
```